pkg/ebpf/gen.go

```diff
@@ -0,0 +1,4 @@
+package ebpf
+
+// $BPF_CLANG and $BPF_CFLAGS are set by the Makefile.
+//go:generate bpf2go -cc $BPF_CLANG -cflags $BPF_CFLAGS -target amd64,arm64,ppc64le,s390x -type flow_metrics_t -type flow_id_t -type flow_record_t -type pkt_drops_t -type dns_record_t -type global_counters_key_t -type direction_t -type filter_action_t -type tcp_flags_t Bpf ../../bpf/flows.c -- -I../../bpf/headers
```

Comment:
I understand we need this for the bpf2go to work but we ended up with this emoty file now so what is really the motivation for renaming ebpf pkg to traces ?

Reply:
`ebpf` package is a dependency of `model`, since it contains the internal data model.
`tracer.go` brings in a lot more dependencies that we don't want to be pulled by `model`.